tests/test_housing_price.py

```diff
@@ -2,5 +2,12 @@
 
 
 def test_module():
-
-    import house_package_saachi
\ No newline at end of file
+    try:
+        from src.house_package_saachi import data_ingestion
+        from src.house_package_saachi import model_scoring
+        from src.house_package_saachi import model_training
+        from src.house_package_saachi import utils
+        from src.house_package_saachi import config
+        from src.house_package_saachi import data_preprocessing

```

Comment:
Here again, all the _imports_ must be like `from house_package_saachi.data_ingestion import load_housing_data` and not `from src.house_package_saachi.data_ingestion import load_housing_data`

Reply:
Changed the test file accordingly. Removed src